cmd/lncli/commands.go

```diff
@@ -1069,9 +1109,12 @@ func sendPayment(ctx *cli.Context) error {
 			}
 		}
 
+		feeLimit = calculateFeeLimit(ctx, amount)
+
 		req = &lnrpc.SendRequest{
-			Dest: destNode,
-			Amt:  amount,
+			Dest:     destNode,
+			Amt:      amount,
+			FeeLimit: feeLimit,
```

Comment:
I think the FeeLimit may need to be computed in the struct on line 1072 as well (in the case where a payment request is being fulfilled).

Reply:
Since it is entirely possible to use the `pay_req` without specifying the amount, I think the fee limit calculating probably should be done in the RPC server, after the pay req is parsed.